ee/clickhouse/views/cohort.py

```diff
@@ -35,6 +48,63 @@ def insert_cohort_people_into_pg(cohort: Cohort):
     cohort.insert_users_list_by_uuid(items=[str(id[0]) for id in ids])
 
 
+def insert_cohort_actors_into_ch(cohort: Cohort, filter_data: Dict):
+    insight_type = filter_data.get("insight")
+
+    if insight_type == INSIGHT_TRENDS:
+        filter = Filter(data=filter_data)
```

Comment:
Problem: This doesn't simplify, meaning test account filters don't get accounted for. It probably should.